app/views/greetings/hello.html.erb

```diff
@@ -1,7 +1,7 @@
-<head>
+<% content_for :head do %>
   <title>Home - Wye RetroApp</title>
   <link href="https://fonts.googleapis.com/css?family=Cabin" rel="stylesheet">
-</head>
+<% end %>
```

Comment:
Since I ended up changing `content_for :head` to an instance variable named `@title`, I won't add the `title` tag in any view. 
Therefore I'll add the font to the CSS file as you mentioned :)